pyvista/core/pointset.py

```diff
@@ -1054,3 +1093,39 @@ def _get_attrs(self):
         attrs = PointGrid._get_attrs(self)
         attrs.append(("Dimensions", self.dimensions, "{:d}, {:d}, {:d}"))
         return attrs
+
+    def hide_cells(self, ind):
```

Comment:
Why is this on `StructuredGrid` and the removal method on `PointSet`? It might even make sense to merge the two adding a `hide=False` parameter.

Reply:
I discovered through trial and error that you can't hide cells in an `UnstrucutredGrid` or `PolyData`, but you can in a `StructuredGrid`.  I'd prefer having the same functionality with both the classes, but that's not the way `vtk` is built.

Reply:
That definitely deserves a comment - such obscure information tends to come back to bite you and then such a comment may save quite a bit of time.

Reply:
Added within the method.